lib/LogHelper.js

```diff
@@ -11,6 +11,8 @@
                 'devDebug': 'LIFE_MODE=DEBUG',
                 'fullDevDebug': 'LIFE_MODE=FULL_DEBUG'
             };
+            // use ERR as a default level
+            this.logLevel = 4;
```

Comment:
This number is a log level that is currently represents DEBUG. we need to read this value from LIFE_MODE